src/main/java/com/aws/greengrass/status/FleetStatusService.java

```diff
@@ -568,8 +612,12 @@ private void uploadFleetStatusServiceData(Set<GreengrassService> greengrassServi
                 .messageType(MessageType.fromTrigger(trigger))
                 .deploymentInformation(deploymentInformation)
                 .build();
-
-        publisher.publish(fleetStatusDetails, components);
+        try {
+            Thread.sleep(10_000); // Prevent messages sent too frequently and cloud received them out of order
```

Comment:
how does this prevent frequent message publish?

Reply:
Before sending each message, sleep for 10 seconds to make sure the previous message is sent already

Reply:
Updated